apko-snapshot/action.yaml

```diff
@@ -130,6 +130,17 @@ inputs:
       In subsequent steps, if automount-src is found, create a copy at this location (inside container)
     default: /work
 
+  source-date-epoch:
+    description: |
+      The UNIX timestamp to use as the source date when building an image.
+    default: ''
```

Comment:
We also allow providing this input to `apko-build` and `apko-publish.`
I'm also interested in it. So, taking the opportunity to add it.
Let me know if it makes sense to keep it here.